backend/api/Areas/Contacts/Models/Search/ContactFilterModel.cs

```diff
@@ -0,0 +1,95 @@
+using Pims.Core.Extensions;
+using Pims.Dal.Entities.Models;
+using System;
+using System.Collections.Generic;
+
+namespace Pims.Api.Areas.Contact.Models.Search
+{
+    public class ContactFilterModel : PageFilter
+    {
+        #region Properties
```

Comment:
q: The properties/constructors/methods seem somewhat obvious to me. What is the team convention around these?

Reply:
Previous convention was to write comments for everything and often write comments that just restate the name of the variable. I hate that convention, prefer that comments are only written when they provide value. Would recommend we move towards that.

Reply:
Oh, i meant the #region tags

Reply:
I prefer to provide comments only where they would add value; e.g. to clarify something, or document a design/dev decision. Happy to move towards less repetitive comments

Reply:
Oh, if you want to change the region tags either log a ticket or schedule a meeting, this is an established pattern that I would not change as part of this story.